docs/source/manim_directive.py

```diff
@@ -150,29 +152,13 @@ def run(self):
         else:
             ref_block = ""
 
-        frame_rate = 30
-        pixel_height = 480
-        pixel_width = 854
-
         if "quality" in self.options:
-            quality = self.options["quality"]
-            if quality == "low":
-                pixel_height = 480
-                pixel_width = 854
-                frame_rate = 15
-            elif quality == "medium":
-                pixel_height = 720
-                pixel_width = 1280
-                frame_rate = 30
-            elif quality == "high":
-                pixel_height = 1440
-                pixel_width = 2560
-                frame_rate = 60
-            elif quality == "fourk":
-                pixel_height = 2160
-                pixel_width = 3840
-                frame_rate = 60
-
+            quality = f'{self.options["quality"]}_quality'
+        else:
+            quality = "test_quality"
```

Comment:
What is the purpose of naming it "test_quality"?
As this is the quality that is used by default in the web examples, why not name it "web_example_quality"

Reply:
Oh. Right. `example_quality` does make more sense!

Reply:
As this is something user-facing, I think `example_quality` could lead to some confusion, as they might not know what this is used for.
Therefore, my preferred way would be to name it `web_example_quality` which is a bit more explicit

Reply:
But I don't have a too strong opinion on this.

Reply:
Done